gravitee-am-model/src/main/java/io/gravitee/am/model/scim/Address.java

```diff
@@ -90,6 +90,10 @@ public Boolean isPrimary() {
         return primary;
     }
 
+    public Boolean getPrimary() {
```

Comment:
curious, why do need `getPrimary` when we already have `isPrimary` method? Same thing you did for Attributes.java class as well?